app/views/bgeigie_imports/index.html.erb

```diff
@@ -51,6 +51,17 @@
 <%- if @bgeigie_imports.empty? -%>
   <%= t("none_found") -%>
 <%- else -%>
+
+  <ul class="inline pull-right">
+    <%-if user_signed_in? %>
+      <%- if current_user.moderator? -%>
+        <li id="open_all" data-bgeigie="<%=@bgeigie_imports.to_json%>">
+          <%=link_to t(".Open all"),params, :class => 'btn' %>
```

Comment:
Can probably have this be a link to `"#"` instead of `params` and just avoid the navigation altogether if JS isn't enabled.